src/components/MoviesList/MoviesList.tsx

```diff
@@ -4,12 +4,26 @@ import { MovieCard } from '../MovieCard';
 
 interface Props {
   movies: Movie[];
+  query: string;
 }
 
-export const MoviesList: React.FC<Props> = ({ movies }) => (
-  <div className="movies">
-    {movies.map(movie => (
-      <MovieCard key={movie.imdbId} movie={movie} />
-    ))}
-  </div>
-);
+const filtered = (moviesList: Movie[], query: string) => {
+  return moviesList.filter(e => {
+    const lowerTitle = e.title.toLocaleLowerCase();
+    const lowerQuery = query.toLocaleLowerCase();
+
+    return lowerTitle.includes(lowerQuery);
```

Comment:
also should be checking description
![Screenshot from 2023-03-03 19-14-43](https://user-images.githubusercontent.com/33029864/222784823-52d41252-723d-472f-8383-65634cea2f04.png)